vhdbuilder/packer/components.json

```diff
@@ -238,8 +244,11 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:*",
       "amd64OnlyVersions": [
         "v0.6.0",
+        "v0.7.9",
         "v1.0.2",
-        "v1.1.1"
+        "v1.0.6",
+        "v1.1.1",
+        "v1.1.2"
```

Comment:
why a new image tag is added. (v1.1.2)

Reply:
latest version 1.1.2 is in section multiArchVersions, pipeline added it to multiArchVersions as we will not touch multiArchVersions for now